.idea/events-service.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe it could be nice to exclude the `.idea/` files? :small_airplane:

Reply:
Oh, my bad those files came.. I will remove them with the coming commits